lib/inspec/resources/auditd.rb

```diff
@@ -102,8 +102,14 @@ def parse_content
     end
 
     def file_syscall_syntax_rules_for(line)
+      # Actions and lists can be in either order
+      valid_actions = %w{never always}
+      action_list = action_list_for(line)
+
+      action = (action_list & valid_actions)
+      list = (action_list - valid_actions)
```

Comment:
Look like there is a failing test, because these both end up as Arrays, rather than the expected String (presumably the first element in the remaining Array).

```

1) Failure:
--
  | Inspec::Resources::AuditDaemon#test_0006_check auditd file watch with syscall syntax [/workdir/test/unit/resources/auditd_test.rb:50]:
  | expected | 0s
  | actual | 1s
  | @@ -1 +1 @@
  | -[{"file"=>"/usr/bin/chage", "list"=>"exit", "action"=>"always", "fields"=>["path=/usr/bin/chage", "perm=x", "auid>=1000", "auid!=-1", "key=privileged"], "permissions"=>["x"], "key"=>"privileged", "fields_nokey"=>["path=/usr/bin/chage", "perm=x", "auid>=1000", "auid!=-1"]}]
  | +[{"file"=>"/usr/bin/chage", "list"=>["exit"], "action"=>["always"], "fields"=>["path=/usr/bin/chage", "perm=x", "auid>=1000", "auid!=-1", "key=privileged"], "permissions"=>["x"], "key"=>"privileged", "fields_nokey"=>["path=/usr/bin/chage", "perm=x", "auid>=1000", "auid!=-1"]}]
  |  


```

Reply:
`make_my_diffs_pretty!` really helps with these types of failures...

A change like this probably needs to be driven via a failing test. I'm also concerned that the change went into `file_syscall_syntax_rules_for` but not `syscall_rules_for`. I would assume that since they both call `action_list_for` that they'd both be affected.